packages/backend/src/config/logger.ts

```diff
@@ -0,0 +1,44 @@
+import { createLogger, format, transports } from 'winston'
```

Comment:
Well, maybe logger.ts should be placed in another folder like utils, or shared?

Reply:
Yeah, it could indeed. It is however a config for the winston instance that we will use. It defines how it should rotate files, where it should store them, etc.

Will see if I come up with a better option. If not, I'd go for moving it inside an utils directory, sure.